_examples/fruits_test.go

```diff
@@ -59,10 +65,31 @@ func TestFruits(t *testing.T) {
 	obj.Value("colors").Array().Elements("green", "red")
 	obj.Value("colors").Array().Element(0).String().Equal("green")
 	obj.Value("colors").Array().Element(1).String().Equal("red")
+	obj.Value("colors").Array().Element(1).String().IsASCII()
+	obj.Value("colors").Array().Element(1).String().HasPrefix("re")
+	obj.Value("colors").Array().Element(1).String().HasSuffix("ed")
 
 	obj.Value("weight").Number().Equal(200)
 
 	e.GET("/fruits/melon").
 		Expect().
 		Status(http.StatusNotFound)
+
+	apple["image"] = []map[string]string{{"id": " 1", "url": "http://example.com", "type": "fruit"}, {"id": "2", "url": "http://example2.com", "type": "fruit"}}
+
+	e.PUT("/fruits/apple").WithJSON(apple).
+		Expect().
+		Status(http.StatusNoContent).NoContent()
```

Comment:
We do all PUTs in the beginning of the test. Let's move this one there too?

Also, it doesn't need to be a separate PUT. You can modify `apple` declaration at the beginning of the test and include `image` there.

Also, second GET wont be needed, we already have `obj` with retrieved apple.

Reply:
👍🏽